wfd/public/video_format.h

```diff
@@ -147,6 +150,25 @@ struct H264VideoFormat {
   RateAndResolution rate_resolution;
 };
 
+struct SupportedH264VideoFormats {
```

Comment:
Got it, thanks! could you maybe add a short dedicated comment to the code pls (saying that it's spec-driven)